documentation/docs/quick-start.md

```diff
@@ -758,30 +759,37 @@ automaticIssuance is optional. It can also be manually triggered and confirmed b
 </Tabs>
 
 
-<summary>Code examples</summary>
+#### Code examples
 
-5. The exchange between CredentialOffer and CredentialRequest is demonstrated through more advanced code samples below, showcasing how different platforms handle it.
+5. The exchange between `CredentialOffer` and `CredentialRequest` is demonstrated through more advanced code samples below, showcasing how different platforms handle it.
 
 <Tabs>
 <TabItem value="js" label="Typescript">
 
 ```typescript
-props.agent.addListener(ListenerKey.MESSAGE,async (newMessages:SDK.Domain.Message[]) => {
-    //newMessages can contain any didcomm message that is received, including
-    //Credential Offers, Issued credentials and Request Presentation Messages
-    const credentialOffers = newMessages.filter((message) => message.piuri === "https://didcomm.org/issue-credential/2.0/offer-credential");
-
-    if (credentialOffers.length) {
-        for(const credentialOfferMessage of credentialOffers) {
-        const credentialOffer = OfferCredential.fromMessage(credentialOfferMessage);
-        const requestCredential = await props.agent.prepareRequestCredentialWithIssuer(credentialOffer);
-        try {
-            await props.agent.sendMessage(requestCredential.makeMessage())
-        } catch (err) {
-            console.log("continue after err", err)
-        }
-        }
+agent.addListener(SDK.ListenerKey.MESSAGE, async (newMessages: SDK.Domain.Message[]) => {
+  // newMessages can contain any didcomm message that is received, including
+  // Credential Offers, Issued credentials and Request Presentation Messages
+  const credentialOffers = newMessages.filter(
+    (message) => message.piuri === 'https://didcomm.org/issue-credential/3.0/offer-credential'
```

Comment:
Proposing to change 'https://didcomm.org/issue-credential/3.0/offer-credential' to SDK.ProtocolType.DidcommOfferCredential